src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1996,3 +2051,23 @@ impl DdlController {
             .await
     }
 }
+
+fn report_create_object<'a>(
+    catalog_id: u32,
+    get_info_fn: impl Fn() -> (
+        &'a str,
+        PbTelemetryDatabaseObject,
+        Option<String>,
+        Option<jsonbb::Value>,
+    ),
```

Comment:
Does it make any difference than just passing the 4 parameters? If no, I prefer keeping it simple.